src/_guides/language/effective-dart/style.md

```diff
@@ -362,34 +356,34 @@ Doing so avoids the [dangling else][] problem.
 
 [dangling else]: http://en.wikipedia.org/wiki/Dangling_else
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (curly-braces)"?>
 {% prettify dart %}
-if (true) {
-  print('sanity');
+if (isWeekDay) {
```

Comment:
The analyzer was (rightfully) reporting dead code. To avoid having to turn that check off, I changed the example.